res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1846,7 +1847,11 @@ PioneerDDJSX.wheelLeds = function(value, group, control) {
         }
         PioneerDDJSX.wheelLedsBlinkStatus[deck]++;
     }
-    PioneerDDJSX.wheelLedControl(group, wheelPos);
+    wheelPos = parseInt(wheelPos);
+    if(PioneerDDJSX.wheelLedsPosition[deck] !== wheelPos){
```

Comment:
```suggestion
    if (PioneerDDJSX.wheelLedsPosition[deck] !== wheelPos) {
```

Reply:
Also please add a comment here explaining why this is necessary.

Reply:
Done

Reply:
Please add a comment too explaining why this check is necessary.